low_level_platform/impl/src/lf_zephyr_support.c

```diff
@@ -154,6 +154,40 @@ void initialize_lf_thread_id() {
 
 int lf_thread_id() { return *((int*)k_thread_custom_data_get()); }
 
+lf_thread_t lf_thread_self() { return k_current_get(); }
+
+int lf_thread_set_cpu(lf_thread_t thread, int cpu_number) { return k_thread_cpu_pin(thread, cpu_number); }
+
+/**
+ * Real-time scheduling API
+ */
+int lf_thread_set_priority(lf_thread_t thread, int priority) {
+  k_thread_priority_set(thread, LF_SCHED_MAX_PRIORITY - priority);
+  return 0;
+}
+
+int lf_thread_set_scheduling_policy(lf_thread_t thread, lf_scheduling_policy_t* policy) {
+  // Update the policy
+  switch (policy->policy) {
+    break;
+  case LF_SCHED_TIMESLICE: {
+    k_thread_priority_set(thread, LF_SCHED_MAX_PRIORITY - policy->priority);
+    k_sched_time_slice_set(0, policy->time_slice / MSEC(1));
+    break;
+  }
+  case LF_SCHED_PRIORITY: {
+    k_thread_priority_set(thread, 99 - policy->priority);
```

Comment:
Should this `99` be `LF_SCHED_MAX_PRIORITY`?

Reply:
I have fixed this now